frontend/src/bundles/common/pages/wallet-details/wallet-details.tsx

```diff
@@ -1,73 +1,171 @@
+import { ButtonSize } from '~/bundles/common/enums/button-size.enum.js';
+import { ButtonVariant } from '~/bundles/common/enums/button-variant.enum.js';
+import { CardVariant } from '~/bundles/common/enums/card-variant.enum.js';
+import { useCallback, useState } from '~/bundles/common/hooks/hooks';
+import { mockSliderData } from '~/bundles/common/pages/dashboard/mocks.dashboard';
+import { type DataType } from '~/bundles/common/types/dropdown.type';
 
-import { FaIcons } from '../../enums/enums';
-import { InputType } from '../../enums/input-type.enum';
-import { useAppForm } from '../../hooks/hooks';
+import { FaIcons } from '../../enums/enums.js';
+import { InputType } from '../../enums/input-type.enum.js';
+import { useAppForm } from '../../hooks/hooks.js';
 import styles from './styles.module.scss';
 
 const DEFAULT_INPUT: { note: string } = {
     //It needs to change
     note: '',
 };
 
+const people = [
+    {
+        value: 'John Doe',
+        name: 'John Doe',
+        image: 'https://placekitten.com/50/50',
+    },
+    {
+        value: 'Jane Smith',
+        name: 'Jane Smith',
+        image: 'https://placekitten.com/51/51',
+    },
+    {
+        value: 'Alice Johnson',
+        name: 'Alice Johnson',
+        image: 'https://placekitten.com/52/52',
+    },
+    {
+        value: 'Bob Brown',
+        name: 'Bob Brown',
+        image: 'https://placekitten.com/53/53',
+    },
+    {
+        value: 'Charlie Green',
+        name: 'Charlie Green',
+        image: 'https://placekitten.com/54/54',
+    },
+];
+
+const categories = [
+    // props to Doughnut Chart
+    {
+        value: 'shopping',
+        name: 'Shopping',
+        image: 'https://img.freepik.com/free-photo/girl-holds-fashion-shopping-bag-beauty_1150-13673.jpg',
+    },
+    {
+        value: 'food',
+        name: 'Food',
+        image: 'https://media.istockphoto.com/id/153010865/uk/%D1%84%D0%BE%D1%82%D0%BE/%D1%87%D0%B5%D1%80%D0%B2%D0%BE%D0%BD%D0%B5-%D1%8F%D0%B1%D0%BB%D1%83%D0%BA%D0%BE.jpg?s=612x612&w=0&k=20&c=Hr4iJ9Taaiaz4SLEAX7BbFSvTsnSrgg-KlRV4IIHG4s=',
+    },
+    {
+        value: 'games',
+        name: 'Games',
+        image: 'https://img.freepik.com/free-vector/joystick-game-sport-technology_138676-2045.jpg?w=2000',
+    },
+    {
+        value: 'car',
+        name: 'Car',
+        image: 'https://store-images.microsoft.com/image/apps.27729.9007199266312409.d86b04ff-7ebb-4451-a920-9cfe5dbd05d7.ffd1ea39-da23-4062-936a-66afe45ffe14?mode=scale&q=90&h=200&w=200&background=%23000000',
+    },
+    {
+        value: 'travel',
+        name: 'Travel',
+        image: 'https://thumbs.dreamstime.com/b/travel-world-landmarks-background-blue-sky-46083021.jpg',
+    },
+];
+
 const WalletDetails: React.FC = () => {
     const { control, errors } = useAppForm<{ note: string }>({
         //It needs to change
         defaultValues: DEFAULT_INPUT,
     });
 
+    const [selectedPeopleMulti, setPeopleSelectedMulti] = useState<
```

Comment:
You should come up with the better name
Same for 98 line

Reply:
fixed